routes/ticketRoutes.js

```diff
@@ -0,0 +1,11 @@
+// API Routes
+const express = require("express");
+const router = express.Router();
+const ticketController = require('../controllers/ticketController')
+router.post('', ticketController.createTicket); // Create a ticket
+router.get('', ticketController.getAllTickets); // Get all tickets
+router.get(':id', ticketController.getTicketById); // Get a single ticket by ID
+router.put(':id', ticketController.updateTicket); // Update a ticket
+router.delete(':id', ticketController.deleteTicket); // Delete a ticket
```

Comment:
> _⚠️ Potential issue_
> 
> **Fix route paths to include leading '/' for parameterized routes**
> 
> The route paths for parameterized routes are missing the leading `'/'`. This could lead to unintended behavior or route mismatches.
> 
> Apply this diff:
> 
> ```diff
> router.post('', ticketController.createTicket); // Create a ticket
> router.get('', ticketController.getAllTickets); // Get all tickets
> -router.get(':id', ticketController.getTicketById); // Get a single ticket by ID
> -router.put(':id', ticketController.updateTicket); // Update a ticket
> -router.delete(':id', ticketController.deleteTicket); // Delete a ticket
> +router.get('/:id', ticketController.getTicketById); // Get a single ticket by ID
> +router.put('/:id', ticketController.updateTicket); // Update a ticket
> +router.delete('/:id', ticketController.deleteTicket); // Delete a ticket
> ```
> 
> 📝 Committable suggestion